src/Command/PullRequest/PullRequestSwitchBaseCommand.php

```diff
@@ -96,26 +95,41 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $gitHelper = $this->getHelper('git');
         $gitHelper->remoteUpdate($sourceOrg);
         $gitHelper->remoteUpdate($pr['base']['user']);
-        $gitHelper->switchBranchBase(
-            $branchName,
-            $pr['base']['user'].'/'.$currentBase,
-            $pr['base']['user'].'/'.$baseBranch,
-            $branchName.'-switched'
-        );
-
-        $gitHelper->pushToRemote($sourceOrg, $branchName.'-switched', true);
-        $gitHelper->pushToRemote($sourceOrg, ':'.$branchName);
-
-        $switchPr = $adapter->switchPullRequestBase(
-            $prNumber,
-            $baseBranch,
-            $sourceOrg.':'.$branchName.'-switched',
-            $input->getOption('force-new-pr')
-        );
 
-        if ($prNumber === $switchPr['number']) {
-            $adapter->createComment($prNumber, sprintf('(PR base switched to %s)', $baseBranch));
```

Comment:
GitHub already informs about this, other adapters can add a comment manually.
All though I'm not sure if the other adapters are fully implemented at the moment.